vignettes/TutorialDaySegmentAnalyses.Rmd

```diff
@@ -119,42 +132,75 @@ GGIR(datadir = "/your/data/directory",
 library("GGIR")
 GGIR(datadir = "/your/data/directory",
              outputdir = "/your/output/directory",
-             mode = 1:2, # <= run GGIR parts 1 and 2
-             do.report = 2, # <= generate csv-report for GGIR part 2
-             qwindow = "/path/to/your/activity/log.csv")
+             mode = 1:5, # <= run GGIR parts 1 to 5
+             do.report = c(2, 5), # <= generate csv-report for GGIR part 2 and part 5
+             qwindow = "/path/to/your/activity/log.csv",
+             timewindow = "MM")
 ```
 
 
-After running this code GGIR creates an output folder in the output directory as specified with argument outputdir.
-In the subfolder `results` you will then find three files:
+After running this code GGIR creates an output folder in the output directory as 
+specified with argument outputdir. In the subfolder `results` you will then find 
+csv files with the reports generated in part 2 and part 5 of the pipeline:
+
+**Part 2**
 
 - `part2_summary.csv` the recording level summary, with 1 row per recording and in recording level aggregates of day segments in columns. 
 - `part2_daysummary.csv` the day level summary, with 1 row per day and day segment specific outcomes in columns.
 - `part2_daysummary_longformat.csv` the day level summary in long format, such that each row represents one segment from one day in one recording.
 
 In both `part2_summary.csv` and `part2_daysummary.csv` the column names tell you the day segment they correspond to. For example, column names ending with `_18-24hr` refer to the time segment 18:00-24:00. In `part2_daysummary_longformat.csv` the time segment is clarified via columns qwindow_timestamps and qwindow_name.
 
+**Part 5**
+
+In part 5, information about the segments of the days are exported in different files
+than the person-level and day-level summaries. These files include the word "Segments"
+in the filename and are provided in the long format and aggregated per day and per person:
+
+- `part5_daysummary_Segments[...].csv` the day level summary in long format, such that each row represents one segment from one day in one recording.
+- `part5_personsummary_Segments[...].csv` the recording level summary in long format, such that each row represents the average for each 
+outcome in one specific segments across all days in which that segment is available per participant.
+
+
+# Cleaning parameters for day segments
+
+The analyses performed per segment of the day come with the possibility to 
+clean the reports based on the information available in the segments. The users
+can select to include only those segments with a given amount of wear time
+during the segment (`segmentWEARcrit.part5`), as well as with a given 
+awake time or sleep period time in the segment (`segmentDAYSPTcrit.part5`). 
```

Comment:
Add sentence: "These arguments are likely to be critical for a meaningful analysis of the data. The presence of sleep in a segment will bias physical activity estimates and the presence of physical activity in a segment will bias sleep in a segment. It will then become impossible to quantify whether it was the lack of one or the presence of the other behaviour that drives the association".